pkg/placement/workload-projector.go

```diff
@@ -1248,7 +1231,7 @@ func (wp *workloadProjector) customizeOrCopy(logger klog.Logger, srcCluster logi
 		if len(refParts) == 1 {
 			custNS = srcObjU.GetNamespace()
 		}
-		customizer, err = wp.customizerClusterLister.Cluster(logicalcluster.Name(srcCluster)).Customizers(custNS).Get(custName)
+		customizer, err = wp.edgeClusterClientset.Cluster(srcCluster.Path()).EdgeV2alpha1().Customizers(custNS).Get(wp.ctx, custName, metav1.GetOptions{})
```

Comment:
Why was this change necessary?

Reply:
I mean, why not fetch the Customzer object from the local cache of an informer watching Customizer objects in the KCS?

Reply:
This seems to have only performance impact, it will take time to change+test and currently the code is working. Opened an issue to re-visit it. 
https://github.com/kubestellar/kubestellar/issues/1329